src/applications/personalization/rated-disabilities/tests/reducers/total-disabilities.unit.spec.js

```diff
@@ -0,0 +1,48 @@
+import { expect } from 'chai';
+
+import { totalRating } from '../../reducers/total-disabilities';
+
+const initialState = {
+  loading: true, // app starts in loading state
+  error: false,
+  totalDisabilityRating: null,
+};
+
+describe('totalDisabilities reducer', () => {
+  it('should return the initial state', () => {
+    const state = totalRating(initialState, {});
+    expect(state.loading).to.equal(true);
+    expect(state.error).to.equal(false);
+    expect(state.totalDisabilityRating).to.equal(null);
+  });
+
+  it('should handle an error from the API call', () => {
+    const state = totalRating(initialState, {
+      type: 'FETCH_TOTAL_RATING_FAILED',
```

Comment:
Probably should import these from the `actions/index.js`.

Reply:
Great feedback, addressed in https://github.com/department-of-veterans-affairs/vets-website/pull/10950/commits/605e2a3d0cfc25f9e941dbf7c81bc6337ea9ef67